PantryNodeReact/src/redux-features/user.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { PayloadAction } from '@reduxjs/toolkit'
+
+const initialState = {
+    name: "",
+    email: "",
+    token: ""
+}
+
+export const userSlice = createSlice({
+    name: 'user',
+    initialState,
+    reducers: {
+        login: (state, action: PayloadAction<{name: string, email: string, token: string}>) => {
```

Comment:
Can you convert this to `createAsyncThunk`? Ideally this involves API calls and thunk can handle it. It's also better to use `extraReducers` with builders for actions which involve API calls

[Reference](https://redux-toolkit.js.org/api/createAsyncThunk)

Reply:
Done